includes/controllers/pages/class-checkout.php

```diff
@@ -175,7 +175,10 @@ private function checkout_form() {
         // $checkout_form .= wpbdp_capture_action( 'wpbdp_checkout_form_top', $this->payment );
         $checkout_form .= $this->gateway->render_form( $this->payment, $this->errors );
         // $checkout_form .= wpbdp_capture_action( 'wpbdp_checkout_form_bottom', $this->payment );
-        $checkout_form .= '<div class="wpbdp-checkout-submit"><input type="submit" value="' . _x( 'Pay Now', 'checkout', 'business-directory-plugin' ) . '" /></div>';
+        $checkout_form .= sprintf(
+            '<div class="wpbdp-checkout-submit"><input type="submit" value="%s" /></div>',
+            $this->payment->show_payment_options() ? esc_html__( 'Pay Now', 'business-directory-plugin' ) : esc_html__( 'Complete', 'business-directory-plugin' )
```

Comment:
Actually... These need to use esc_attr instead of esc_html since it's inside of an HTML attribute.

`$this->payment->show_payment_options() ? esc_attr__( 'Pay Now', 'business-directory-plugin' ) : esc_attr__( 'Complete', 'business-directory-plugin' )`

Reply:
Thanks, this has been updated :)